graf2d/gpadv7/v7/inc/ROOT/RPadDisplayItem.hxx

```diff
@@ -54,6 +56,7 @@ public:
       fFrame = nullptr;
       fDrawOpts = nullptr;
       fSize = nullptr;
+      fWinSize[0] = fWinSize[1] = 0;
```

Comment:
First of all, this is used only for the canvas - I want to avoid too many changes and introduce RCanvasDisplayItem too early.

Second, 0 is special value. It specifies that canvas will be adjusted to size of provided HTML element. And will be auto-adjusted if HTML sizes changed (when browser resized)